corehq/motech/repeaters/models.py

```diff
@@ -560,7 +560,7 @@ class RepeatRecord(Document):
     payload_id = StringProperty()
 
     overall_tries = IntegerProperty(default=0)
-    max_possible_tries = IntegerProperty(default=3)
+    max_possible_tries = IntegerProperty(default=5)
```

Comment:
this will only apply to new repeat records right?

Reply:
Yes.